applications/luci-app-advanced-reboot/root/usr/libexec/rpcd/luci.advanced_reboot

```diff
@@ -66,18 +66,21 @@ get_main_partition_os_info(){
 }
 
 get_alt_partition_os_info(){
-	local op_info op_ubi="$1"
+	local op_info op_ubi="$1" vendor_name="$2" ubi_vol="$3"
 	logger "attempting to mount alternative partition (mtd${op_ubi})"
-	alt_partition_unmount "$op_ubi"
-	alt_partition_mount "$op_ubi"
+	alt_partition_unmount "$op_ubi" "$ubi_vol"
+	alt_partition_mount "$op_ubi" "$ubi_vol"
 	if [ -s "/var/alt_rom/etc/os-release" ]; then
 		op_info="$(. /var/alt_rom/etc/os-release && echo "$PRETTY_NAME")"
 		if [ "${op_info//SNAPSHOT}" != "$op_info" ]; then
 			op_info="$(. /var/alt_rom/etc/os-release && echo "${OPENWRT_RELEASE%%-*}")"
 		fi
 	fi
+	if [ -s "/var/alt_rom/etc/partition_config/soft-version" ]; then
+		op_info="$vendor_name $(awk -F: '$1=="soft_ver" { print $2 ;}' /var/alt_rom/etc/partition_config/soft-version)"
```

Comment:
Maybe `op_info="${vendor_name:+$vendor_name }$(awk -F: '$1=="soft_ver" { print $2 ;}' /var/alt_rom/etc/partition_config/soft-version)"` to avoid having a leading space if `$vendor_name` is empty?